internal/simplecue/utils.go

```diff
@@ -177,8 +177,30 @@ func cueConcreteToScalar(v cue.Value) (interface{}, error) {
 
 		return values, nil
 	case cue.StructKind:
-		// TODO: this would be used for struct-level default values
-		return nil, nil //nolint: nilnil
+		newMap := make(map[string]interface{})
```

Comment:
Should we be populating an `orderedmap` here? To guarantee a consistent output of the parser